Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -3750,6 +3751,44 @@ def com_google_fonts_check_integer_ppem_if_hinted(ttFont):
                  " will be rounded into and integer value.")
 
 
+@condition
+def gfonts_repo_structure(font):
+  """ The family at the given font path
+      follows the files and directory structure
+      typical of a font project hosted on
+      the Google Fonts repo on GitHub ? """
+
+  # FIXME: Improve this with more details
+  #        about the expected structure.
+  return font.split("/")[-3] in ["ufl", "ofl", "apache"]
```

Comment:
This won't work for upstream repos.

This also won't work if you nav to a google/fonts family dir e.g /Users/marcfoley/Type/fonts/ofl/montserrat. I get the following traceback.

```
>> com.google.fonts/check/repo/dirname_matches_nameid_1 with (('font[17]', 'Montserrat-ThinItalic.ttf'),)
   Directory name in GFonts repo structure must match NameID 1.
 * ERROR: The condition <FontBakeryCondition:gfonts_repo_structure> had an error: IndexError: list index out of range
           File "/Users/marcfoley/Type/fontbakery/Lib/fontbakery/checkrunner.py", line 377, in _evaluate_condition
             return None, condition(**args)
           File "/Users/marcfoley/Type/fontbakery/Lib/fontbakery/callable.py", line 71, in __call__
             return self._func(*args, **kwds)
           File "/Users/marcfoley/Type/fontbakery/Lib/fontbakery/specifications/googlefonts.py", line 3763, in gfonts_repo_structure
             return font.split("/")[-3] in ["ufl", "ofl", "apache"]
         

   Result: ERROR


Total:

    ERROR: 18
    FAIL: 0
    WARN: 0
    SKIP: 0
    INFO: 0
    PASS: 0

[EEEEEEEEEEEEEEEEEE] 100%

DONE!
```

Reply:
I take it this PR is wip?